z2/resources/chain-specs/zq2-devnet.toml

```diff
@@ -1,6 +1,8 @@
 p2p_port = 3333
 bootstrap_address = [ "12D3KooWFvZum3gpEQkj6o9Tg71uLuQZJ4hqXL32c8wPTcpkRnaP", "/dns/bootstrap.zq2-devnet.zilliqa.com/tcp/3333" ]
 
+otlp_collector_endpoint = "http://localhost:4317"
```

Comment:
I don't think we want this. Its up to validators if they want to collect metrics. We shouldn't assume they will (nor that their collector will be running on port 4317).

@pavlops @frankmeds Thoughts?

Reply:
yes, let's add this as z2 join param...... if this is required in the config.toml

Reply:
Sure I can do that now